lib/mix/lib/mix/release.ex

```diff
@@ -206,6 +206,18 @@ defmodule Mix.Release do
     """)
   end
 
+  defp erts_lib_dir(erts_path) when is_function(erts_path) do
+    erts_lib_dir(erts_path.())
+  end
+
+  defp erts_lib_dir(erts_path) when is_binary(erts_path) or is_list(erts_path) do
+    Path.join([erts_path, "..", "lib"])
+    |> Path.expand()
+    |> to_charlist()
```

Comment:
I think this can be written as: `erts_path |> Path.dirname |> Path.join("lib") |> to_charlist`.